src/preprocess.rs

```diff
@@ -97,6 +93,80 @@ pub fn process_chapter(
         .join("")
 }
 
+/// When `pre-render` is called but not enabled.
+#[cfg(not(feature = "pre-render"))]
+pub fn process_all_chapters_prerender(
+    _: &Vec<String>,
+    _: &KatexConfig,
+    _: &str,
+    _: &PreprocessorContext,
+) -> Vec<String> {
+    panic!("Unable to pre-render. Please rebuild with the feature `pre-render`!")
```

Comment:
Suggestion: suggest the users to use escape mode and provide a link to the README section.